pkgs/applications/editors/eclipse/build-eclipse.nix

```diff
@@ -30,6 +30,7 @@
 
 stdenv.mkDerivation rec {
   inherit name src;
+  version = productVersion;
```

Comment:
don't set `name` and set `pname` instead https://nixos.org/manual/nixpkgs/unstable/#sec-using-stdenv

https://github.com/NixOS/rfcs/pull/35

Reply:
ah, thanks -- done!